test/unit/models/mappers/check-activities.test.js

```diff
@@ -23,7 +23,7 @@ describe('Check Activity Mappers', () => {
         event: {
           activity: {
             activityType: 'received',
-            activityLabel: 'Police correspondence'
+            activityLabel: 'Police correspondence received'
```

Comment:
A more realistic test is if the label is say 'Police correspondence from Bedforshire Police received'